apps/server/src/models/groups.ts

```diff
@@ -29,7 +29,7 @@ class Groups extends Model<
   declare id: string
   declare title: string
   declare description: string
-  declare adminId: string
+  declare userId: string
```

Comment:
The db design says adminId. Why change it to userId?? I believe adminId is what we settled for when refactoring the Groups model. Kindly make the changes, thank you.